### PR TITLE
Collect vars and add beta toggles for theme editor

### PR DESCRIPTION
**Pull request by Inwerpsel**
This adds the build script to make the CSS variables file used for the theme editor, as well as a toggle which will be used by https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/495

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "bootstrap": "^4.3.1",
     "classnames": "^2.2.6",
     "jquery": "^3.5.1",
-    "popper.js": "^1.15.0"
+    "popper.js": "^1.15.0",
+    "postcss-css-variables-extract": "^0.1.2"
```
Comment (Inwerpsel) [inline package.json:40]: Probably I should move this to the dev dependencies if we only need it during build

File: src/Features.php [hunk 2]
```diff
@@ -66,6 +70,24 @@ private static function get_fields(): array {
...
+				'name' => __( 'Theme editor (experimental)', 'planet4-master-theme-backend' ),
+				'desc' => __(
+					'Enable CSS variables based theme editor for logged in users. Do not use in production yet.',
+					'planet4-master-theme-backend'
+				),
+				'id'   => self::THEME_EDITOR,
+				'type' => 'checkbox',
+			],
+			[
+				'name' => __( 'Theme editor non-logged in(experimental)', 'planet4-master-theme-backend' ),
```
Comment (Inwerpsel) [inline src/Features.php:83]: I made this a separate checkbox to be sure people see the warning about enabling it for all users.

Comment (comzeradd) [reply src/Features.php:83]: I left a comment on the other PR. I think it would make more sense not to enable that for non-logged-in users.

**Comment by comzeradd** [comment]
Since you are on the Features page, can you make a small css tweak to make sure the option text stays on the same line?

fearures

Looks like this line is causing the problem:

```css
.cmb-th+.cmb-td {
    float: left;
}
```

**Comment by Inwerpsel** [comment]
> Since you are on the Features page, can you make a small css tweak to make sure the option text stays on the same line?
> 
> fearures
> 
> Looks like this line is causing the problem:
> 
> ```css
> .cmb-th+.cmb-td {
>     float: left;
> }
> ```

Added this fix https://github.com/greenpeace/planet4-master-theme/blob/de89e006b98c7ad97b55b855e3c3223479d0fe0a/admin/css/campaign.css#L37-L39

**Review by comzeradd — APPROVED** [review]

**Pull request merged**